frameworks/g-cloud-10/questions/declaration/termsOfParticipation.yml

```diff
@@ -1,6 +1,6 @@
 name: Invitation to apply
 question: >
-  Do you agree to comply with the terms of the <a href="/suppliers/frameworks/g-cloud-10" target="_blank" rel="noopener noreferrer">G-Cloud&nbsp;10 ‘invitation to apply’ (link opens in a new tab)</a>?
+  Do you agree to comply with the terms of the <a href="/suppliers/frameworks/g-cloud-10#guidance" target="_blank" rel="noopener noreferrer">G-Cloud&nbsp;10 ‘invitation to apply’ (link opens in a new tab)</a>?
```

Comment:
Is this linking to the right place? Seems like weird question - can't tell where it should be going.

Reply:
Yeah it links to guidance. When we upload the invitation to apply, it appears under guidance.